### PR TITLE
Intake | Add missing non-rating issue categories

### LINKED ISSUE #15276: Intake | Add missing non-rating issue categories
## User or job story
User story: As an intake user, I would like to see some additional non-rating issue categories, so that I don't intake these as unidentified issues when I don't see them:

**Categories** (copy not final):
- Compensation
  - DIC
  - SC Death
  - Accrued benefits
- Pension
  - NSC live pension
  - Survivor’s pension
  - Accrued benefits

## Acceptance criteria
- [ ] Please add the above categories to ISSUE_CATEGORIES.json

**Note**: this does not need to be behind a feature toggle.

## Open questions
- What benefit type is each issue category?
- Official copy

## Release notes
New non-rating issue categories have been added for Compensation and Pension.

### Background/context
These were discovered during missing ratings research.

### PR DESCRIPTION
**Pull request by wdrougas-tista**
Resolves #15276 

### Description
Non rating issue categories added to ISSUE_CATEGORIES.json

### CODE REVIEW

**Review by jcq — APPROVED** [review]
Looks like it added the specified categories!

One thing that's a bit odd is the capitalization... some of these are title case, some are sentence case... I feel like it'd be good to have a standard? Might be worth checking w/ @rutvigupta-design, but also probably nobody cares

**Comment by jcq** [comment]
Oh, and it looks like you'll need to update `decision_reviews.yaml` to match (see failing test)

**Review by meckilabritt-navapbc — APPROVED** [review]

File: client/constants/ISSUE_CATEGORIES.json [hunk 1]
```diff
@@ -27,11 +27,13 @@
   "compensation": [
```
Comment (meckilabritt-navapbc) [inline client/constants/ISSUE_CATEGORIES.json:27]: It looks like the rspec that checks categories just needs to be updated!

**Comment by rutvigupta-design** [comment]
Adding my 2 cents here - if possible and not too much effort, let's standardize to Title Case for now, since I think that's what already exists. I have...general thoughts about this, but like JC said, no one probably cares :)

File: client/constants/ISSUE_CATEGORIES.json [hunk 2]
```diff
@@ -63,11 +67,13 @@
...
     "Income/Net Worth | Medical Expense Deductions",
     "Income/Net Worth | Residential Lot Size",
+    "NSC live pension",
     "Overpayment | Validity of debt",
     "Overpayment | Waiver",
     "Penalty Period",
     "Post Award Audit",
-    "Survivors pension eligibility"  
+    "Survivor's pension",
+    "Survivor's pension eligibility"
```
Comment (leikkisa) [inline client/constants/ISSUE_CATEGORIES.json:76]: I hope updating these a little doesn't impact reporting, I can't see Tableau reports but I don't think there's a ton of reporting by non-rating issue category.  We may want to do an update on the data in prod to update existing categories to the new version.

I think it doesn't hurt to ask before merging this.

**Review by leikkisa — COMMENTED** [review]
It looks like the API team has a spec that uses this YAML file `caseflow/app/controllers/api/docs/v3/decision_reviews.yaml` so you may need to update that file as well with the new categories.

I think this looks good, but since there's no rush on this ticket, it might be good to ask the data group about changing the existing non-rating issue category.

**Comment by wdrougas-tista** [comment]
I will go ahead and update to Title Case across categories but will wait to merge if that works @rutvigupta-design @leikkisa

**Comment by leikkisa** [comment]
Hey @wdrougas-tista - I got some input from Yoom on what to do if something _might_ impact existing reporting.

Since there's no rush on this PR, I think it makes sense to notify the reporting team (see below). If we don't hear back after a few days, I'd say go ahead and move forward with the PR since both Yoom and I don't know of any impacts.

You can create a ticket at:
https://github.com/department-of-veterans-affairs/dsva-vacols/issues

And let them know what we're changing (it's okay to link the PR), and ask if this would adverse affect any of their reports.  Yoom said you can also tag ` @TedM-BVA` who leads the reporting team.

**Comment by leikkisa** [comment]
Just linking Will's request here: https://github.com/department-of-veterans-affairs/dsva-vacols/issues/153

**Pull request merged**